Gruntfile.js

```diff
@@ -195,37 +202,30 @@ module.exports = function( grunt ) {
 	} );
 
 	grunt.registerTask( "lint", [
-		"jsonlint",
-		runIfNewNode( "eslint:dev" ),
-		runIfNewNode( "eslint:dist" )
+		"newer:jsonlint",
+		runIfNewNode( "newer:eslint:dev" ),
+		runIfNewNode( "newer:eslint:dist" )
 	] );
 
-	grunt.registerTask( "test_fast", [ runIfNewNode( "node_smoke_tests" ) ] );
```

Comment:
Since I'm not entirely understand why this is convenient, can I get another "+1" for someone for brining this back?

Would like to land this tomorrow